pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +281,64 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+func TestApplyObjectOutput(t *testing.T) {
+	initTestErrorHandler(t)
+	nameRC, currentRC := readAndAnnotateReplicationController(t, filenameRC)
+	pathRC := "/namespaces/test/replicationcontrollers/" + nameRC
+
+	// Add some extra data to the post-patch object
+	postPatchObj := &unstructured.Unstructured{}
+	if err := json.Unmarshal(currentRC, &postPatchObj.Object); err != nil {
+		t.Fatal(err)
+	}
+	postPatchLabels := postPatchObj.GetLabels()
+	if postPatchLabels == nil {
+		postPatchLabels = map[string]string{}
+	}
+	postPatchLabels["post-patch"] = "value"
```

Comment:
hi, @gyliu513 I think this is testing the update resources, checkout code [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/apply.go#L246), if we testing create a new resource, the return resp should be return `&http.Response{StatusCode: 404...` like [this](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/apply_test.go#L354)